src/screens/RecoverySeed/RecoverySeedScreen.tsx

```diff
@@ -132,10 +133,17 @@ export class RecoverySeedScreen extends Component<Props, State> {
 
     return (
       <ScreenTemplate
+        testID="cancel-seed-phrase-screen"
         header={<Header onBackArrow={onBackArrow} isBackArrow title={i18n.send.recovery.recover} />}
         footer={
           <>
-            <Button loading={isLoading} disabled={!this.canSubmit()} title={buttonText} onPress={this.submit} />
+            <Button
+              testID={'cancel-seed-phrase-cancel-button'}
```

Comment:
Looks weird.

Reply:
`cancel-seed-phrase-button` will be better or `seed-phrase-cancel-button`?

Reply:
done